<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0,0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Definition_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1100</width>
          <height>650</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(158, 205, 255);
font: 22pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
</string>
        </property>
        <property name="text">
         <string>Definition</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>840</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1300</width>
          <height>199</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0,0);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_25">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>25</number>
           </property>
           <item row="1" column="2">
            <widget class="QWidget" name="ans_4" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#ans_4{
    background-color: rgb(207, 164, 255);
    border-radius: 30px;
    padding-left: 25px;
}

#ans_4:hover {
    background-color: rgb(211, 209, 251);
  padding-left: 25px;
}

#ans_4:pressed {
    background-color:rgb(211, 209, 251);
  padding-left: 25px;
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_25">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_25">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>59</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 22pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>D</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>11</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="var_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>Вариант ответа D</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QWidget" name="ans_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#ans_2{
    background-color: rgb(207, 164, 255);
    border-radius: 30px;
 
}

#ans_2:hover {
    background-color: rgb(211, 209, 251);
}
#ans_2:pressed {
    background-color:rgb(211, 209, 251);
}

var_2{font: 12pt &quot;Comic Sans MS&quot;;}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>56</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 22pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>B</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>11</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="var_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string extracomment="0">Вариант ответа B</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QWidget" name="widget_2" native="true">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QWidget" name="ans_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>75</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#ans_3{
    background-color: rgb(207, 164, 255);
    border-radius: 30px;
    padding-left: 25px;
}

#ans_3:hover {
    background-color: rgb(211, 209, 251);
  padding-left: 25px;
}
#ans_3:pressed {
    background-color:rgb(211, 209, 251);
  padding-left: 25px;
}

var_3{font: 12pt &quot;Comic Sans MS&quot;;}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_24">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_23">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>54</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 22pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>C</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>11</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="var_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>Вариант ответа C </string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QWidget" name="ans_1" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>75</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#ans_1{
    background-color: rgb(207, 164, 255);
    border-radius: 30px;
    padding-left: 25px;
}

#ans_1:hover {
    background-color: rgb(211, 209, 251);
  padding-left: 25px;
}
#ans_1:pressed {
    background-color:rgb(211, 209, 251);
  padding-left: 25px;
}

var_1{font: 12pt &quot;Comic Sans MS&quot;;}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 22pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>A</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>11</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="var_1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
                </property>
                <property name="text">
                 <string>Вариант ответа А</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/icons.qrc"/>
 </resources>
 <connections/>
</ui>
